eiffel-vocabulary/EiffelEnvironmentDiscontinuedEvent.md

```diff
@@ -0,0 +1,19 @@
+
+# EiffelEnvironmentDiscontinuedEvent (EnvD)
```

Comment:
EnvDisc is probably a better abbreviation since EnvD can be interpreteded as EiffelEnvironmentDefinedEvent.